packages/protocol/contracts/common/GasPriceMinimum.sol

```diff
@@ -126,7 +124,7 @@ contract GasPriceMinimum is
       uint256 rateNumerator;
       uint256 rateDenominator;
       (rateNumerator, rateDenominator) = sortedOracles.medianRate(tokenAddress);
-      return (gasPriceMinimum.mul(rateNumerator).div(rateDenominator));
+      return (gasPriceMinimum * rateNumerator) / rateDenominator;
```

Comment:
I think there are many places where SafeMath is redundant, but are we sure this is one of them? `rateNumerator` can be influenced by Oracles, and could be arbitrarily large.